### DISCUSSION TITLE
android端，setPathAsync 缓存错误问题

### DISCUSSION THREAD

**Discussion by peter100u** [Tencent/libpag#2106] [General]
版本 4.3.47, 在上个版本，把我们自己的下载逻辑，全部换成pag自带的下载和缓存逻辑，如
 pagView.setPathAsync(levelEffectsUrl) { file ->

        if (isIllegalPagFile(levelEffectsUrl, file)) {
            return@setPathAsync
        }

        pagView.composition = file
        pagView.setRepeatCount(0)
        pagView.play()
    }

但是在使用的时候，发现，在app内大量的使用这个方法，在不同的页面，极端情况下会出现，资源错位的情绪，比如A pagview上加载的内容却是B的pagview的file, B的pagview上却是其他资源的情况，赶紧像是异步情况下，没有根据url和pagview id锁定在一起的感觉， 由于是极端情况下，而且我们是语音产品，基本每个页面都是pagview/pagimage 所以我给不出demo,

**Comment by kevingpqi123** [comment]
目前其它产品还没有反馈过类似问题，仅仅靠上面的描述获取的信息非常有限，建议基于 pag 的官方 demo 做下测试，看下能否复现出你们遇到的问题

**Reply by peter100u** [reply]
嗯，我知道，但是很难必现出来

**Comment by CodeJhF** [comment]
之前请你补充的复现信息（demo / 版本号 / 机型 / 日志）一直没有收到，缺少这些信息无法继续定位，先关闭本条。
如果问题还在，欢迎补齐上述信息后回复或重开，我们会优先跟进。（stale-close）
